skrf/vectorFitting.py

```diff
@@ -1156,6 +1304,170 @@ def plot_s_mag(self, i: int, j: int, freqs: Any = None, ax: mplt.Axes = None) ->
         ax.set_title('Response i={}, j={}'.format(i, j))
         return ax
 
+    @check_plotting
+    def plot_s_deg(self, i : int, j: int, freqs: Any = None, ax: mplt.Axes = None) -> mplt.Axes:
```

Comment:
I wonder if the plotting methods should be moved to `plotting.py`. Could you evaluate if this is a lot of work?

Reply:
To be honest, the plotting module seems like a wild beast to me; I don't really get how it all works out. It also seems to be tightly coupled to the `Network` class, so it would be very different from the `VectorFitting` class. For example, the plotting functions in `VectorFitting` will work for arbitrary frequencies, i.e. outside the frequencies specified in the `Network` that was fitted.
So, I would prefer to leave them separate in this module. If, for some reason, it really needs to get merged into `plotting.py`, we should probably do it in another PR.

Reply:
OK, let's keep like that for the moment.